omero/developers/Model/EveryObject.txt

```diff
@@ -4,14 +4,6 @@ Glossary of all OMERO Model Objects
 Overview
 --------
 
-In navigating the model objects used by the :doc:`OMERO API
```

Comment:
Ideally we'll someday have code that, rather than just outputting a new file, optionally reads an existing one, copies that to the output, and just replaces the autogenerated segment of it.